sui_programmability/examples/fungible_tokens/sources/regulated_coin.move

```diff
@@ -98,13 +98,13 @@ module abc::abc {
     use sui::transfer;
     use std::vector;
 
-    /// The ticker of ABC regulated token
-    struct ABC has drop {}
+    /// The ticker of Abc regulated token
+    struct Abc has drop {}
```

Comment:
amazing lol

Reply:
It uses `Abc` instances beyond the initializer, but if @damirka can rewrite this code for `Abc` to be a one-time witness type, then we can revert this!

Reply:
No this is fine, I just found it funny that we had to change it